src/amo/components/Addon/index.js

```diff
@@ -474,6 +499,7 @@ export class AddonBase extends React.Component {
     return (
       <div
         className={makeClassName('Addon', `Addon-${addonType}`, {
+          'Addon-theme': this.addonIsTheme(),
```

Comment:
Do we have a test case for this? (or simply an assertion)

Reply:
I will add.. thanks